app/Http/Controllers/Admin/DataMapCrudController.php

```diff
@@ -108,8 +108,8 @@ protected function setupCreateOperation()
                         'wrapper' => ['class' => 'form-group col-md-4'],
                     ],
                     [
-                        'name' => 'label',
-                        'label' => 'Label',
+                        'name' => 'column_name',
```

Comment:
This really isn't enough for someone to know what should go here. The crud controller has references to the soils project with information that is not relevant here; the 'model' variable should not ask about a single data table that the form will populate, as datamaps are per-module, not per-form. 

See my latest pull request for a revised version with detailed labels.